bikeshed/spec-data/readonly/bikeshed-version.txt

```diff
@@ -1 +1 @@
-Bikeshed version 5fcd28d6d, updated Tue May 30 13:12:11 2023 -0700
```

Comment:
I don't understand why this says 2023 while https://github.com/speced/bikeshed/commit/0474eae152d6c8f278bd9dbdd76449415c71484c clearly has it in 2024?!